packages/mds-policy-service/tests/policy-repository.spec.ts

```diff
@@ -323,6 +322,10 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
         ...metadata,
         policy_metadata: { name: 'steve' }
       })
+      const changedMetadata = (await PolicyRepository.readSinglePolicyMetadata(
+        ACTIVE_POLICY_JSON.policy_id
+      )) as PolicyMetadataDomainModel<{ name: string }>
```

Comment:
Can use the generic now instead of a typecast!

Reply:
it's good!